docs/developers/config_tin.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+from enum import Enum
+
+
+class ArmoryMode(Enum):
```

Comment:
I feel like these are actually two different things.  Also, assuming this file would be in path like `from armory.environment import *` I would use something like:

```
class ExecutionMode(Enum):
  HOST = "host"
  DOCKER = "docker"

class AccessMode(Enum):
  ONLINE = "online"
  OFFLINE = "offline"
```

Reply:
yes, these should be disjoint as you suggest.